agents/node/vcxagent-core/src/utils/vcx-workflows.js

```diff
@@ -49,32 +54,43 @@ async function provisionAgentInAgency (agentName, genesisPath, agencyUrl, seed,
   if (!seed) {
     throw Error('seed not specified')
   }
-  const provisionConfig = {
-    agency_url: agencyUrl,
-    agency_did: 'VsKV7grR1BUE29mG2Fm2kX',
-    agency_verkey: 'Hezce2UWMZ3wUhVkh2LfKSs8nDzWwzs2Win7EzNN3YaR',
+
+  const walletConfig = {
     wallet_name: agentName,
     wallet_key: '123',
-    payment_method: 'null',
-    enterprise_seed: seed
+    wallet_key_derivation: 'ARGON2I_INT'
```

Comment:
Let's use `RAW` with key `8dvfYSt5d1taSd6yJdpjq4emkwsPDDLYxkNFysFD2cZY` (these can be generated with function provided by libindy). That should make tests a little bit faster.